### PR TITLE
TII-194 update allowed TII file types

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/TII-194

TII has documentation describing the allowed file types, which contains types that we didn't previously list.

The following sakai.properties are involved here (if you wish to override the defaults):

turnitin.acceptable.mime.types
turnitin.acceptable.file.extensions
turnitin.acceptable.file.types

Documentation of allowed file types can be found here:

https://guides.turnitin.com/01_Manuals_and_Guides/Student/Student_User_Manual/09_Submitting_a_Paper

Also, on the student's submission page, if the assignment is set to send submissions to Turnitin, the (long) list of allowed file types is duplicated. This should only be presented to the user once.

### CODE REVIEW

**Comment by bjones86** [comment]
This PR is directly related to https://github.com/sakaicontrib/turnitin/pull/18

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm [hunk 1]
```diff
@@ -747,7 +733,18 @@ $(document).ready(function(){
 						#else
 							$tlang.getString("gen.att")
 						#end
-					</h3>	
+					</h3>
+					#if ($assignment.getContent().getAllowReviewService())
+						<p class="instruction"><img alt="$reviewIndicator" title="$reviewIndicator" src="/library/image/silk/rosette.png" /> $reviewIndicator</p>
```
Comment (buckett) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: I don't think this change should have come back on the 11.x branch as it's not using the new Sakai 11 icons.

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm [hunk 2]
```diff
@@ -716,18 +713,7 @@ $(document).ready(function(){
 						#end
 					</p>
 				#end	
-				#if (($submissionType == 2) || ($submissionType == 5))
-					#if ($plagiarismNote)
-						<p>
-							<i alt="$reviewIndicator" title="$reviewIndicator" class="fa fa-certificate reviewStyle"></i>
-							<em>
-								$plagiarismNote
```
Comment (buckett) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: What's the reason for removing `$plagiarismNote` here?

Comment (bjones86) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: Prior to this change, the `$plagiarismNote` was a very lengthy string which also contained valid file types. As file types are not relevant to inline submissions, this PR changes things around a bit.

Now, `$plagiarismNote` refers only to inline submissions, and does not contain anything about file types. It simply states "Note: Your inline submission will be sent to {0} to be electronically reviewed for plagiarism.". As such, this message should only be presented (above the CKEditor) when the type of assignment is either 1 or 3 (inline or attachments and inline).

The file type warning which used to be duplicated on the page (one above the CKEditor and one below), is now only presented if the assignment type is 2, 3 or  (attachments only, attachments and inline, or single file upload).

Comment (buckett) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: Thanks for the good explanation, should it also be removed from `AssignmentAction`?

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm (hunk 1 above)
Comment (bjones86) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: Yes, you are correct on this one. I'll amend this in another PR.

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm (hunk 2 above)
Comment (bjones86) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: Possible, I'll look into that for the PR to come. Thanks @buckett!

Comment (bjones86) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: Not necessary to remove from `AssignmentAction`, it's still used in the Velocity template on lines 646 and 650.

**Pull request merged**